mobile/src/modules/auth/components/terms/TermsModal.tsx

```diff
@@ -48,11 +53,12 @@ export class TermsModal extends React.PureComponent<Props> {
     const { visible } = this.props;
 
     return (
-      <Modal
-        animationType={'slide'}
-        visible={visible}
-        transparent={false}
-      >
+      // This used to be a modal but has a freezing glitch - blankey
+      // <Modal
+      //   animationType={'slide'}
+      //   visible={visible}
+      //   transparent={false}
+      // >
```

Comment:
This fix centers around the Terms Modal no longer being a modal anymore. But I left this here to explain why its still called TermsModal in the code...

Reply:
what about renaming the component itself? - Keeping commented code feels wrong